ajax/libs/flot.tooltip/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "flot.tooltip",
+  "filename": "jquery.flot.tooltip.min.js",
+  "version": "0.8.5",
+  "description": "Tooltip plugin for Flot plotting library",
+  "homepage": "https://github.com/krzysu/flot.tooltip",
+  "keywords": [
+    "flot",
+    "tooltip",
+    "chart",
+    "graph"
+  ],
+  "repositories": [
+    {
+      "type": "git",
+      "url": "https://github.com/krzysu/flot.tooltip"
+    }
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/krzysu/flot.tooltip.git",
+    "basePath": "js",
+    "files": [
+      "jquery.flot.tooltip.js",
+      "jquery.flot.tooltip.min.js"
```

Comment:
In the same directory there is

* a bunch of unneeded files (at least for 0.8.5, it was cleaned later), which are unversioned copies of Flot library at some point of time
* `jquery.flot.tooltip.source.js`, that is a `jquery.flot.tooltip.js` without attribution (which is added by some build script).